src/ui/locales/ar.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "تبديل الإحالة",
+    "AttributionControl.MapFeedback": "ملاحظات الخريطة",
```

Comment:
Might need some context here to give a better translation but I'm assuming this is some sort of title for a section with the feedback about the map. If its intended for a button with a user action like "leave feedback" then this text would be different. I couldn't figure out that context from the english translation either.
```suggestion
    "AttributionControl.MapFeedback": "تعليقات على الخريطة",
```

Reply:
Hey @oielbanna 👋 

Thanks for helping us, I've posted more context in this PR https://github.com/mapbox/mapbox-gl-js/pull/11733

Reply:
Happy to help! 
Reading this, I would say use "علق على الخريطة" because then that uses a verb (hence an action btn) and it roughly translates to "Give feedback on the map".